### DISCUSSION TITLE
troika-three-text: Does `getSelectionRects` respect `letterSpacing` of spaces?

### DISCUSSION THREAD

**Discussion by lockinmarv** [protectwise/troika#258] [Q&A] [answered]
Hey! I'm manually getting the width of letters/words within the whole text, but once I add a `letterSpacing`, the size of the space literally never changes and always stays at the number of `9.27557373046875` (or very similar).

Letter spacing tests:
0.2 → 9.27557373046875
0    → 9.275566101074219
0.5 → 9.27557373046875

Is this intended, because without the space it's impossible to get the correct size? 

Thanks!

**Comment by lojjic** [comment]
Looks like a bug, thanks for reporting!
<img width="772" alt="image" src="https://github.com/protectwise/troika/assets/138124/15dc4839-a80d-4006-90ea-ef3b7d8d304d">

**Comment by lojjic** [comment, marked as answer]
Created issue #259

**Reply by lockinmarv** [reply]
Thanks for creating the issue! 🙏
